sample-configurations/foak-fast-kernels-sample-configuration.yaml

```diff
@@ -22,10 +22,13 @@ plugins:
       # - the FastQuantized version is all-or-nothing
 
       # fast loss triton kernels
-      fast_loss: True
+      fast_loss: False
```

Comment:
the config must be properly installed in the benchmark utilty ,.we cannot do hardcoding like this otherwise all the other benches will be affected.